src/UserStories/UserStories.js

```diff
@@ -113,40 +155,34 @@ export function UserStories({ userId }) {
       <div
         ref={allStoriesContainer}
         className="stories-scrollable"
-        style={{ transform: `translateX(-${scrollAmount}px)` }}
+        style={{ transform: `translateX(${-scrollAmount}px)` }}
       >
         {Object.keys(storyIndices).length &&
           storiesData.map((user) => {
-            const activeStory = userId === user.user_id;
-            const story = activeStory ? activeUserStory : user.stories[0];
+            const activeUser = userId === user.user_id;
+            const story = activeUser ? activeUserStory : user.stories[0];
```

Comment:
A little out of scope, but I'd a comment here
```suggestion
            // when the user isn't specified, we go to the first story
            const story = activeUser ? activeUserStory : user.stories[0];
```

Reply:
Done